service-02/meta.json

```diff
@@ -114,6 +114,10 @@
             {
               "from": "Dummy",
               "types": "DummyType"
+            },
+            {
+              "from": "Parameter",
```

Comment:
Parameter is in common - in what other service it will be used?

Reply:
3 and 4

Reply:
ack

Reply:
I moved it from service 3, because service 4 uses it